src/boost_fractor_transformer.jl

```diff
@@ -284,11 +289,14 @@ function transformer(bdry::SetupBoundaries, coords::CoordinateSystem, modes::Mod
 
     initial = emit
     #println(initial)
-
-    axion_beam = Array{Complex{T}}(zeros((modes.M)*(2modes.L+1)))
-
+    axion_beam = Array{Complex{Float64}}(zeros((modes.M)*(2modes.L+1)))
```

Comment:
I should probably also try and include this dynamic typing into my code. I think this was added after i started writing but it seems like a good idea.

Reply:
The dynamic typing here had some specific reason I am not entirely aware of (Jacob needed it).
In general you are right, this would be better style. But it also makes the code less readable imo. I would in general leave it for now.